### ISSUE TITLE
Pre-work Checklist: Developer: Chris Menke

### ISSUE THREAD

**Issue by chrismenke45** [hackforla/website#3828]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [ ] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by chrismenke45** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
Onboarding meeting: 2.75hrs
Tuesday HFLA Website meeting: 1.75hrs
Reviewing (someone else's) a pull request: 1hr
Selecting/finding my good first issue: .25hrs
Solving first issue(including environment set up and git challenges): 2 hours

**Comment by ExperimentsInHonesty** [comment]
@chrismenke45 Good job on the first, second, small and medium issues.  And that you have self-assigned a Large issue.  You also got the hang of doing updates.  So I am going to close this issue as completed.  

Hope you will be joining the merge team soon!  See this document for details on how to move your journey forward: https://github.com/hackforla/website/wiki/Being-a-Part-of-the-Hack-For-LA--Dev-Team

FYI @hackforla/website-merge

**Comment by HackforLABot** [comment]
## Activity Log: chrismenke45
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- chrismenke45 opened: https://github.com/hackforla/website/issues/3828 at 01/16/2023, 07:47 PM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/3828 at 01/16/2023, 08:05 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/3835#issuecomment-1386464635 at 01/17/2023, 08:18 PM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/3835#pullrequestreview-1253896785 at 01/18/2023, 08:49 AM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/2868#issuecomment-1048471373 at 01/18/2023, 09:10 AM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/2868#issuecomment-1387432271 at 01/18/2023, 09:12 AM PST
- chrismenke45 opened: https://github.com/hackforla/website/pull/3841 at 01/18/2023, 03:03 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3828#issuecomment-1396249209 at 01/18/2023, 03:59 PM PST
- chrismenke45 PR was merged: https://github.com/hackforla/website/pull/3841#event-8327008568 at 01/21/2023, 12:43 AM PST
- chrismenke45 issue closed by PR 3841: https://github.com/hackforla/website/issues/2868#event-8327008592 at 01/21/2023, 12:43 AM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/3399 at 01/21/2023, 08:18 AM PST
- chrismenke45 opened: https://github.com/hackforla/website/pull/3847 at 01/21/2023, 08:42 AM PST
- chrismenke45 PR was merged: https://github.com/hackforla/website/pull/3847#event-8339587200 at 01/23/2023, 04:55 PM PST
- chrismenke45 issue closed by PR 3847: https://github.com/hackforla/website/issues/3399#event-8339587345 at 01/23/2023, 04:55 PM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/3283#issuecomment-1306296320 at 01/26/2023, 07:27 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3283#issuecomment-1406938188 at 01/27/2023, 10:47 AM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3283#issuecomment-1411150837 at 01/31/2023, 02:24 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3283#issuecomment-1421991109 at 02/07/2023, 08:39 PM PST
- chrismenke45 issue closed as completed: https://github.com/hackforla/website/issues/3283#event-8494438317 at 02/10/2023, 03:03 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/3961#issuecomment-1430343772 at 02/14/2023, 12:38 PM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/3961#pullrequestreview-1298399441 at 02/14/2023, 12:50 PM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/2625#issuecomment-1407299488 at 02/14/2023, 12:56 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1430369172 at 02/14/2023, 12:58 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1430493696 at 02/14/2023, 02:48 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/3977#issuecomment-1430705721 at 02/14/2023, 07:40 PM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/3977#pullrequestreview-1299875043 at 02/15/2023, 08:39 AM PST
- chrismenke45 opened: https://github.com/hackforla/website/pull/3984 at 02/15/2023, 09:28 AM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/3982#pullrequestreview-1300051309 at 02/15/2023, 09:43 AM PST
- chrismenke45 PR was merged: https://github.com/hackforla/website/pull/3984#event-8555686624 at 02/19/2023, 09:50 AM PST
- chrismenke45 issue closed by PR 3984: https://github.com/hackforla/website/issues/2625#event-8555686659 at 02/19/2023, 09:50 AM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4011#issuecomment-1437700370 at 02/20/2023, 04:31 PM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4011#pullrequestreview-1306457182 at 02/20/2023, 04:36 PM PST
- chrismenke45 assigned: https://github.com/hackforla/website/issues/3985 at 02/20/2023, 04:38 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1437704544 at 02/20/2023, 04:40 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4022#issuecomment-1439394627 at 02/21/2023, 07:39 PM PST
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4022#pullrequestreview-1309726781 at 02/22/2023, 08:58 AM PST
- chrismenke45 issue closed as completed: https://github.com/hackforla/website/issues/3828#event-8609055952 at 02/25/2023, 11:30 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1447291552 at 02/27/2023, 03:44 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1449290406 at 02/28/2023, 07:50 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1464558902 at 03/10/2023, 02:27 PM PST
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4135#issuecomment-1465254039 at 03/12/2023, 10:27 AM PDT
- chrismenke45 assigned: https://github.com/hackforla/website/issues/3744#issuecomment-1463381616 at 03/12/2023, 11:19 AM PDT
- chrismenke45 unassigned: https://github.com/hackforla/website/issues/3985#issuecomment-1464558902 at 03/12/2023, 11:19 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4135#pullrequestreview-1337776826 at 03/13/2023, 12:12 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3744#issuecomment-1466950654 at 03/13/2023, 02:01 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3744#issuecomment-1474506290 at 03/17/2023, 04:20 PM PDT
- chrismenke45 opened: https://github.com/hackforla/website/pull/4280 at 03/24/2023, 12:56 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3744#issuecomment-1483340620 at 03/24/2023, 12:58 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3744#issuecomment-1483568945 at 03/24/2023, 04:19 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1483593225 at 03/24/2023, 04:39 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4308#issuecomment-1487875894 at 03/28/2023, 07:56 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4308#pullrequestreview-1363297032 at 03/29/2023, 07:43 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4347#issuecomment-1490650224 at 03/30/2023, 10:12 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4347#pullrequestreview-1365598918 at 03/30/2023, 10:30 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4347#issuecomment-1490716370 at 03/30/2023, 11:07 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/3744#issuecomment-1496291746 at 04/04/2023, 09:46 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1498435851 at 04/05/2023, 08:17 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1498472254 at 04/05/2023, 09:19 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499191037 at 04/06/2023, 07:48 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499447528 at 04/06/2023, 11:18 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499759578 at 04/06/2023, 04:38 PM PDT
- chrismenke45 issue closed by PR 4280: https://github.com/hackforla/website/issues/3744#event-8956718228 at 04/07/2023, 05:20 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4468#issuecomment-1504284464 at 04/11/2023, 04:51 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1504460330 at 04/11/2023, 07:31 PM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4483 at 04/11/2023, 08:04 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4468#pullrequestreview-1381683741 at 04/12/2023, 09:26 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4468#issuecomment-1506141773 at 04/12/2023, 05:32 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1514069891 at 04/18/2023, 08:06 PM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4517#event-9048817043 at 04/19/2023, 09:46 AM PDT
- chrismenke45 closed by PR 4517: https://github.com/hackforla/website/issues/4448#event-9048817330 at 04/19/2023, 09:46 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4497#event-9048828183 at 04/19/2023, 09:47 AM PDT
- chrismenke45 closed by PR 4497: https://github.com/hackforla/website/issues/4445#event-9048828389 at 04/19/2023, 09:47 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4524#event-9048942563 at 04/19/2023, 10:00 AM PDT
- chrismenke45 closed by PR 4524: https://github.com/hackforla/website/issues/4383#event-9048942822 at 04/19/2023, 10:00 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4518#event-9049025522 at 04/19/2023, 10:09 AM PDT
- chrismenke45 closed by PR 4518: https://github.com/hackforla/website/issues/4422#event-9049025729 at 04/19/2023, 10:09 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4545#pullrequestreview-1392976760 at 04/19/2023, 03:43 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4468#issuecomment-1517896750 at 04/21/2023, 07:10 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4554#issuecomment-1518438309 at 04/21/2023, 05:15 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1518438640 at 04/21/2023, 05:16 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4554#issuecomment-1519289252 at 04/23/2023, 07:18 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1519289567 at 04/23/2023, 07:18 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4554#pullrequestreview-1398510469 at 04/24/2023, 10:35 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1522039222 at 04/25/2023, 08:54 AM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4567 at 04/25/2023, 05:52 PM PDT
- chrismenke45 assigned: https://github.com/hackforla/website/issues/4530 at 04/25/2023, 07:05 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1522677022 at 04/25/2023, 07:33 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4474#pullrequestreview-1402785573 at 04/26/2023, 01:40 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4561#pullrequestreview-1402799942 at 04/26/2023, 01:50 PM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4561#event-9107826025 at 04/26/2023, 01:51 PM PDT
- chrismenke45 closed by PR 4561: https://github.com/hackforla/website/issues/4414#event-9107826237 at 04/26/2023, 01:51 PM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4558#event-9107834388 at 04/26/2023, 01:52 PM PDT
- chrismenke45 closed by PR 4558: https://github.com/hackforla/website/issues/4394#event-9107834579 at 04/26/2023, 01:52 PM PDT
- chrismenke45 unassigned: https://github.com/hackforla/website/issues/4530 at 04/26/2023, 02:01 PM PDT
- chrismenke45 closed as completed: https://github.com/hackforla/website/issues/4567#event-9158778296 at 05/03/2023, 11:36 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4567#issuecomment-1533523810 at 05/03/2023, 11:39 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4680#pullrequestreview-1442301823 at 05/24/2023, 09:15 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4680#event-9330015040 at 05/24/2023, 09:15 AM PDT
- chrismenke45 closed by PR 4680: https://github.com/hackforla/website/issues/4375#event-9330015387 at 05/24/2023, 09:15 AM PDT
- chrismenke45 assigned: https://github.com/hackforla/website/issues/4538 at 05/30/2023, 08:56 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4538#issuecomment-1568688077 at 05/30/2023, 08:58 AM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4756 at 05/30/2023, 09:20 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4538#issuecomment-1568723744 at 05/30/2023, 09:21 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4756#issuecomment-1568724364 at 05/30/2023, 09:22 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1569403110 at 05/30/2023, 07:24 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4745#pullrequestreview-1453429089 at 05/31/2023, 07:47 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4745#event-9391605786 at 05/31/2023, 07:47 AM PDT
- chrismenke45 closed by PR 4745: https://github.com/hackforla/website/issues/4491#event-9391606197 at 05/31/2023, 07:47 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4748#pullrequestreview-1453433290 at 05/31/2023, 07:49 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4748#event-9391637565 at 05/31/2023, 07:49 AM PDT
- chrismenke45 closed by PR 4748: https://github.com/hackforla/website/issues/4419#event-9391638269 at 05/31/2023, 07:49 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4754#pullrequestreview-1453469246 at 05/31/2023, 08:03 AM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4754#event-9391820285 at 05/31/2023, 08:03 AM PDT
- chrismenke45 closed by PR 4754: https://github.com/hackforla/website/issues/3998#event-9391820662 at 05/31/2023, 08:03 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4474#pullrequestreview-1453529481 at 05/31/2023, 08:22 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4538#issuecomment-1572377068 at 06/01/2023, 09:35 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1575668353 at 06/04/2023, 11:32 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1577686671 at 06/05/2023, 04:32 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4764#pullrequestreview-1465432536 at 06/06/2023, 08:03 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4538#issuecomment-1579759266 at 06/06/2023, 07:09 PM PDT
- chrismenke45 closed as completed: https://github.com/hackforla/website/issues/4538#event-9452651323 at 06/06/2023, 07:09 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4755#issuecomment-1580809226 at 06/07/2023, 06:20 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1580823991 at 06/07/2023, 06:27 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1595938725 at 06/17/2023, 08:27 PM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4764#pullrequestreview-1485203840 at 06/18/2023, 02:58 PM PDT
- chrismenke45 assigned: https://github.com/hackforla/website/issues/4794 at 06/19/2023, 08:02 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4794#issuecomment-1597345764 at 06/19/2023, 08:03 AM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4864 at 06/20/2023, 09:12 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4864#issuecomment-1599102462 at 06/20/2023, 09:13 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4794#issuecomment-1599103480 at 06/20/2023, 09:13 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4794#issuecomment-1603215419 at 06/22/2023, 12:40 PM PDT
- chrismenke45 assigned: https://github.com/hackforla/website/issues/4769 at 06/27/2023, 04:56 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1610379583 at 06/27/2023, 04:58 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1613530900 at 06/29/2023, 09:55 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4890#issuecomment-1613535211 at 06/29/2023, 09:59 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/pull/4897#issuecomment-1613543848 at 06/29/2023, 10:05 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4897#pullrequestreview-1505764498 at 06/29/2023, 10:15 AM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4903 at 06/30/2023, 03:36 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4903#issuecomment-1615250418 at 06/30/2023, 03:36 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1615251047 at 06/30/2023, 03:38 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1616249486 at 07/01/2023, 06:04 PM PDT
- chrismenke45 closed as completed: https://github.com/hackforla/website/issues/4769#event-9698192090 at 07/01/2023, 06:04 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4903#issuecomment-1616730238 at 07/02/2023, 09:54 AM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1618933370 at 07/03/2023, 10:41 AM PDT
- chrismenke45 submitted review: https://github.com/hackforla/website/pull/4911#pullrequestreview-1515263527 at 07/05/2023, 12:55 PM PDT
- chrismenke45 merged: https://github.com/hackforla/website/pull/4911#event-9735853515 at 07/05/2023, 12:55 PM PDT
- chrismenke45 closed by PR 4911: https://github.com/hackforla/website/issues/2288#event-9735853720 at 07/05/2023, 12:55 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1622406004 at 07/05/2023, 12:56 PM PDT
- chrismenke45 opened: https://github.com/hackforla/website/issues/4912 at 07/05/2023, 01:18 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4912#issuecomment-1622464392 at 07/05/2023, 01:36 PM PDT
- chrismenke45 unassigned: https://github.com/hackforla/website/issues/4769 at 07/10/2023, 05:06 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1629899672 at 07/10/2023, 05:06 PM PDT
- chrismenke45 closed as completed: https://github.com/hackforla/website/issues/4912#event-9780996210 at 07/10/2023, 05:07 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4903#issuecomment-1629900519 at 07/10/2023, 05:08 PM PDT
- chrismenke45 closed by PR 5298: https://github.com/hackforla/website/issues/4903#event-9781000374 at 07/10/2023, 05:08 PM PDT
- chrismenke45 commented on: https://github.com/hackforla/website/issues/4912#issuecomment-1629900573 at 07/10/2023, 05:08 PM PDT
- chrismenke45 issue closed as completed: https://github.com/hackforla/website/issues/4794#event-9889338504 at 07/21/2023, 03:26 PM PDT
